### PR TITLE
Bump Javacord version to 3.0.1

### PR DESCRIPTION
**Pull request by Saladoc**

### CODE REVIEW

File: src/main/java/org/javacord/bot/commands/GradleCommand.java [hunk 1]
```diff
@@ -21,7 +21,7 @@ public void onCommand(TextChannel channel) {
                           "  mavenCentral()\n" +
                           "}\n" +
                           "dependencies { \n" +
-                          "  implementation 'org.javacord:javacord:3.0.0'\n" + // TODO Always use the latest version
+                                  "  implementation 'org.javacord:javacord:3.0.1'\n" + // TODO Always use the latest version
```
Comment (Vampire) [inline src/main/java/org/javacord/bot/commands/GradleCommand.java]: Why the indentation change?

File: src/main/java/org/javacord/bot/commands/MavenCommand.java [hunk 2]
```diff
@@ -20,7 +20,7 @@ public void onCommand(TextChannel channel) {
                            "<dependency>\n" +
                            "    <groupId>org.javacord</groupId>\n" +
                            "    <artifactId>javacord</artifactId>\n" +
-                           "    <version>3.0.0</version>\n" + // TODO Always use the latest version
+                                  "    <version>3.0.1</version>\n" + // TODO Always use the latest version
```
Comment (Vampire) [inline src/main/java/org/javacord/bot/commands/MavenCommand.java]: Why the indentation change?

File: src/main/java/org/javacord/bot/commands/GradleCommand.java (hunk 1 above)
Comment (Vampire) [inline src/main/java/org/javacord/bot/commands/GradleCommand.java]: While not using the latest Javacord version from some request, at least use `Javacord.VERSION` in the meantime? But keep the TODO or better make an issue instead or additionally.

File: src/main/java/org/javacord/bot/commands/MavenCommand.java (hunk 2 above)
Comment (Vampire) [inline src/main/java/org/javacord/bot/commands/MavenCommand.java]: While not using the latest Javacord version from some request, at least use `Javacord.VERSION` in the meantime? But keep the TODO or better make an issue instead or additionally.

File: src/main/java/org/javacord/bot/commands/GradleCommand.java (hunk 1 above)
Comment (Saladoc) [reply src/main/java/org/javacord/bot/commands/GradleCommand.java]: Formatter issues.

**Pull request merged**